src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/BlackDuckPolicyLinkableItem.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.provider.blackduck.collector;
+
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import com.synopsys.integration.alert.common.message.model.LinkableItem;
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+import com.synopsys.integration.alert.provider.blackduck.BlackDuckProviderContentTypes;
+
+public class BlackDuckPolicyLinkableItem extends AlertSerializableModel {
+    private final SortedSet<LinkableItem> linkableItems;
```

Comment:
Do we know what these LinkableItems are here? We could be more explicit if so (e.g. policyRuleLinks or something like that).